packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdInSummary.spec.ts

```diff
@@ -82,21 +90,67 @@ describeCompat("Create data store with group id", "NoCompat", (getTestObjectProv
 		groupId: string | undefined,
 		message: string,
 	) => {
-		assert(snapshotTree.omitted, message);
 		assert(snapshotTree.groupId === groupId, message);
 		assert(Object.entries(snapshotTree.trees).length === 0, message);
 		assert(Object.entries(snapshotTree.blobs).length === 0, message);
 	};
 
+	const assertOmittedBlobContents = (
```

Comment:
This function should replace assertOmittedTree

Reply:
Actually, both are testing different things. In assertOmittedTree(), the trees and blobs actually empty and that is when we fetched the grouped snapshot.